packages/manager/modules/support/src/tickets/new-ticket/issues-form/resources/resources.html

```diff
@@ -0,0 +1,29 @@
+<oui-tile>
```

Comment:
Don't know if it can be the case, but what if `$ctrl.guides` and `$ctrl.tips` are empty ? `oui-tile` will still be generated. 
```suggestion
<oui-tile data-ng-if="ctrl.guides.length > 0 && $ctrl.tips.length > 0">
```

Reply:
Good point :+1: